sylviawanjiku/StoreManager
# #161914210 Consume all sales endpoints

#### What does this PR do?
Consume sales endpoints
#### Description of Task to be completed?
Have the following endpoints consumed working in the UI
Make a sale
Get all sales record by admin
Get personal sale by attendant
#### How should this be manually tested?
An attendant after making a sale can try to view her sales record and check whether her records can be updated.
An admin can view sales made by any attendant
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#161914210
https://www.pivotaltracker.com/story/show/161914210
#### Screenshots (if appropriate)
#### Questions:
